tests/Countries/SwitzerlandTest.php

```diff
@@ -7,6 +7,42 @@
 use Spatie\Holidays\Exceptions\InvalidRegion;
 use Spatie\Holidays\Holidays;
 
+dataset('cantons', [
```

Comment:
I don't think all cantons need a specific snapshot test. You can only verify the edge cases, which is more explicit.